### DISCUSSION TITLE
Multi-Store

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/commerce#2283] [Ideas] [answered]
- [x] Product types become multi-store/site aware #39 #157 #37 #382
- [x] Variant should also be localized? #167
- [x] Localized (translatable) description of discount/sale/shipping method #199 #457 
- [x] Pricing is per site/ per store #93 
- [x] Store settings become store specific #1681
- [x] Store specific sales #2053

**Comment by pdaleramirez** [comment]
Do you have ETA of the release for this feature? i kinda waiting on this for my project.

**Comment by lukeholder** [comment]
We don't have a specific ETA @pdaleramirez sorry. Won't be in 3.x but maybe will in 4.x

**Comment by pdaleramirez** [comment]
Looking forward to 4.x

**Comment by ghost** [comment]
I'm afraid that although I too am 'looking forward' to this, I seriously cannot wait. I have a project coming up where this is a core requirement, so it looks like I'm going to have to use some other CMS. 

This has been around a long time now. Please please please make sure it makes it into 4.0 ... please ;-)

**Comment by stevenjwright** [comment]
+1 for me on this. Currently handling this with 4 sites in one install (same client company) making use of permissions and product types to separate CP usage and  using some custom reports for orders... not ideal, but making do for now!

**Comment by mariovass** [comment]
+1

**Comment by mark-chief** [comment]
Project in the pipeline for this requirement, really don’t want to use anything else but Craft + Commerce. Thanks.

**Comment by jripmeester** [comment]
+1

**Comment by jordanhammond** [comment]
Choosing an ecommerce CMS now for 4+ website domains using the same ERP. Will require all of the features described in the description. +1 to prioritise this.

**Comment by jordanhammond** [comment]
> I'm afraid that although I too am 'looking forward' to this, I seriously cannot wait. I have a project coming up where this is a core requirement, so it looks like I'm going to have to use some other CMS.
> 
> This has been around a long time now. Please please please make sure it makes it into 4.0 ... please ;-)

@AndrewMac-UTS did you find another CMS product that delivered these requirements?

**Reply by AndrewMac** [reply]
Hi Jordan, I’m so sorry that I missed your post. Please see my reply below. We moved to Drupal.

**Comment by frbron** [comment]
+1

**Comment by ArtDepartmentMJ** [comment]
+1

**Comment by edwardpmartinez** [comment]
+1

**Comment by boboldehampsink** [comment]
+1

**Comment by bhuvidya** [comment]
+1

**Comment by tigerchick** [comment]
That this hasn't yet made it into Commerce seems to be in contradiction to the "core concepts" docs: https://craftcms.com/docs/commerce/3.x/core-concepts.html

"Because Craft Commerce is built on Craft CMS, most Craft features can be integral to the ecommerce experience:
...
- Localization...."

Looking forward to this being addressed.

**Comment by lukeholder** [comment]
This is on the roadmap for 5.0 https://craftcms.com/roadmap

**Reply by jerome2710** [reply]
This is really disappointing, 5.0 feels really far away. We're not even on 4 yet. The roadmap also refers to #2280, which is named multi-currency. This sounds more drastic than multi-lingual. Any chance to separate these two and move localization higher on the list?

**Reply by ArtDepartmentMJ** [reply]
Agreed, this is really disappointing especially since the carrot about it being in 4 was already dangled. We have multiple clients waiting for this functionality and therefore may have to find an alternative product solution.

**Reply by russback** [reply]
+1 

Has always felt like a fundamental gap in a platform that has had great localisation support from day one.

**Reply by boboldehampsink** [reply]
+1

**Reply by jripmeester** [reply]
+1

**Reply by modelesque** [reply]
+1

**Reply by jak-kal** [reply]
> Agreed, this is really disappointing especially since the carrot about it being in 4 was already dangled. We have multiple clients waiting for this functionality and therefore may have to find an alternative product solution.

This. Surely the majority of use cases for multi tenancy would also require multi currency? Such a shame as we're still officially on 3.

**Reply by masiorama** [reply]
+1

**Reply by ArtDepartmentMJ** [reply]
Is this still planned? I can't seem to see it on the roadmap anymore.

**Reply by Nellyaa** [reply]
> Is this still planned? I can't seem to see it on the roadmap anymore.

It's in the "In Progress" section

**Reply by ArtDepartmentMJ** [reply]
> > Is this still planned? I can't seem to see it on the roadmap anymore.
> 
> It's in the "In Progress" section

My bad, I was scouring the Craft roadmap! Glad it's being worked on

**Comment by AndrewMac** [comment]
I'm afraid I had to jump ship. Even though I have used Craft since its release and would have loved to stay with it, I have had to move my client sites over to Drupal, which offers multiple sites, multiple languages, multiple stores and multiple currencies. So I'm no longer sure about the future with Craft. My clients and I put a lot of money into Craft and its third-party developers, but with the delays with this feature, we can't keep doing that. Requirements like this are critical for so many sites—especially, I imagine, once you look beyond the USA—but sadly Craft has given commitments (I hesitate to say 'made promises') and failed to make any progress with any of them. Will I be back? I hope so, but it's looking less and less likely at this stage. And I find that very sad indeed …

**Comment by lukeholder** [comment]
@AndrewMac Commerce 5 beta with Multi-store and inventory management will be out very soon.

**Reply by AndrewMac** [reply]
Hi Luke, and thanks so much for taking the time to post. But don't worry, I've been constantly watching progress on version 5 of Craft and Commerce, and will be back on board pretty quickly, I hope 🥳

**Comment by lukeholder** [comment, marked as answer]
Commerce 5 was just released and has multi-store support! 

https://craftcms.com/commerce
